src/AddTodoForm.js

```diff
@@ -1,15 +1,34 @@
 import React from "react";
 
-function AddTodoForm() {
+
+function AddTodoForm(props) {
+    const handleChange = (event_item) => {
+        // console.log(event_item.target.value)
+        props.onAddTodo(event_item.target.value);
+    };
+
+    function handleAddTodo(events) {
+        events.preventDefault();
+        const todoTitle = events.target.elements.title.value;
+        console.log(todoTitle);
+        props.onAddTodo(todoTitle);
+        events.target.reset();
+    }
+
     return (
-        <form>
+        <form onSubmit={handleAddTodo}>
             <label htmlFor="todoTitle" id="todoTitle">
                 Title
             </label>
-            <input type="text" id="todoTitle"></input>
-            <button>Add</button>
-        </form>
-    );
-}
+            <input
+                type="text"
+                id="todoTitle"
+                name="title"
+                onChange={handleChange}
```

Comment:
why do we have an onChange here? this shouldn't be on this current lesson. right now it's changing the new item whenever you type something. I think we just want the new item to change after clicking on submit.

Reply:
So I should just have on submit? On change is in the reading.

Reply:
I believe `onChange` is on lesson 1.4. Where does it tell you to have onChange in the instructions for 1.3's assignment?

Reply:
I think I got a little confused because it is referenced in one of the videos and the book, but not in the actual lesson text.